pkg/agent/secondarynetwork/podwatch/controller_test.go

```diff
@@ -406,6 +407,7 @@ func TestConfigurePodSecondaryNetwork(t *testing.T) {
 					testIPAMResult("148.14.24.100/24", 101),
 				)
 			},
+			expectedNetworkStatusAnnot: "[{\n    \"name\": \"net\",\n    \"ips\": [\n        \"148.14.24.0\"\n    ],\n    \"dns\": {}\n}]",
```

Comment:
could we also add validation to e2e tests for both VLAN and SRIOV case?

Reply:
Done.